urlsigning/urlSigner.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+  <head>
+    <script src="http://google.github.io/closure-library/source/closure/goog/base.js"></script>
+    <script>
+    goog.require('goog.crypt.Hmac');
+    goog.require('goog.crypt.Sha1');
+    goog.require('goog.crypt.base64');
+    </script>
+  </head>
+  <body>
+    <script>
+    /**
+     * Sign a request URL with a URL signing secret.
+     *
+     * If URL field is empty, output is empty.
+     *
+     * @param{String} inputUrl The URL to be signed including the Client ID/API key.
+     * @param{String} secret The URL signing secret encoded in base64.
+     * @returns{String}
+     */
```

Comment:
Indentation is off. The `/` should line up:

Also, not sure if you'd prefer a named function here.

```js
/**
 * Foo
 */
function signUrl(inputUrl, secret) {
}
```